lib/collection/src/shards/local_shard/mod.rs

```diff
@@ -110,6 +110,8 @@ pub struct LocalShard {
 impl LocalShard {
     /// Moves `wal`, `segments` and `clocks` data from one path to another.
     pub async fn move_data(from: &Path, to: &Path) -> CollectionResult<()> {
+        log::debug!("Moving local shard from {:?} to {:?}", from, to);
```

Comment:
`fmt::Debug` adds quotes around the path, there's `Path::display` method to `fmt::Display` a `Path`.

```suggestion
        log::debug!("Moving local shard from {} to {}", from.display(), to.display());
```